data/extension/init/templates/tinygo/envoy.filters.network/default/main.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
Can we include extension configuration into this init template ?

Reply:
What is the resolution ?

I don't mind what configuration format is.

I'd like to see how `streamContext` instances will get a hold of that configuration.

Reply:
sorry just marked resolved since the comment was same for https://github.com/tetratelabs/getenvoy/pull/104#discussion_r589787169 and just wanted to have the discussion there.